README.rst

```diff
@@ -274,12 +274,43 @@ as devices with the following command.
 
    zenbatchload <filename>
 
+Modeling and Monitoring OS Services
+-----------------------------------
+The Linux OS services are modeled using the *zenoss.cmd.linux.os_service* modeler plugin. The following systems are supported:
+
+- systemd (RHEL 7)
+- upstart (RHEL 6)
+- systemV (RHEL 5 and earlier)
+
+Version 2.3.0 and beyond provides support for monitoring the status of the services every collection cycle. However, only **systemd** and **systemV** are supported for monitoring. The following two zProperties filter the services that are to be modeled and thereby monitored.
+
++---------------------------+------------------------------------------+
+| Name                      | Description                              |
++===========================+==========================================+
+| zLinuxServicesModeled     | Accepts regular expressions that         |
+|                           | matches one or more services to model    |
++---------------------------+------------------------------------------+
+| zLinuxServicesNotModeled  | Accepts regular expressions that         |
+|                           | matches one or more services to not model|
++---------------------------+------------------------------------------+
+
+Only *loaded* services are modeled. By default, both zProperties are ``<empty>``. An ``<empty>`` value in ``zLinuxServiceModeled`` is treated as ``.*`` regex and models all loaded services. Both the zProperties can support multiple regex expressions when separated on new lines. The *OSService* monitoring template generates events on every collection cycle for a service that is down. The events are automatically cleared if the service is up again.
+
```

Comment:
Replace "By default,....bla bla,  when separateed on new lines" with

zLinuxServicesModeled and zLinuxServicesNotModeled are lists that accept regular expressions, and are by default empty.